runtime/autoload/man.vim

```diff
@@ -104,6 +104,9 @@ endfunction
 
 " Run a system command and timeout after 30 seconds.
 function! s:system(cmd, ...) abort
+  " See https://github.com/neovim/neovim/issues/9271
+  unlet $MANSECT
```

Comment:
not sure this is worth doing, see https://github.com/neovim/neovim/issues/9271#issuecomment-557951956

Problem will occur if a colon exists in MANPATH also. See also https://github.com/neovim/neovim/pull/9159#issuecomment-433698704 for details.

Reply:
Removed.